HW_1/task_2.py

```diff
@@ -0,0 +1,24 @@
+"""
+Задача 4: 
+Петя, Катя и Сережа делают из бумаги журавликов. 
+Вместе они сделали S журавликов. 
+Сколько журавликов сделал каждый ребенок, если известно, что 
+Петя и Сережа сделали одинаковое количество журавликов, 
+а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+
+s = x + x + 2*(x + x)
+s = 6*x
+
+Пример:
+6 -> 1  4  1
+24 -> 4  16  4
+60 -> 10  40  10
+"""
+
+print('Введите количество журавликов, которые сделали все дети: ')
+s = int(input())
+
+if s % 6 == 0:    
+    print(f'Петя сделал {s // 6} журавликa(ов), Сережа - {s // 6} журавлика(ов), а Катя - {4 * s // 6} журавлика(ов)')
```

Comment:
У вас повторяется трижды одна и та же операция, соответственно, на каждую такую операцию выделяются ресурсы, чтобы такого не происходило используют переменные.
```suggestion
    var = s // 6
    print(f'Петя сделал {var } журавликa(ов), Сережа - {var} журавлика(ов), а Катя - {4 *var} журавлика(ов)')
```